spec/CloudCodeLogger.spec.js

```diff
@@ -182,6 +182,60 @@ describe('Cloud Code Logger', () => {
     });
   });
 
+  it('should log cloud function triggers using the custom log level', async done => {
```

Comment:
could you refactor this test, it contains two identical blocks; instead you could just write a list of levels and test them in a loop?

Reply:
Commited a fix, should be better now.